deploy/helm/sumologic/values.yaml

```diff
@@ -2534,6 +2545,10 @@ opentelemetry-operator:
     image:
       repository: public.ecr.aws/sumologic/kube-rbac-proxy
 
+  testFramework:
+    image:
+      repository: public.ecr.aws/sumologic/busybox
```

Comment:
What is this?

Reply:
https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/templates/tests/test-service-connection.yaml

Reply:
I thought that was only used for tests. Can it actually appear if we install the helm chart normally?

Reply:
It's generated by `helm template` for the following configuration: https://github.com/SumoLogic/sumologic-openshift-images/blob/0096d8236b323ef3827d2fd91b18c41274560a14/scripts/values.yaml

Reply:
It shouldn't for `helm install`:
```
  annotations:
    "helm.sh/hook": test
```

Reply:
I don't think it really hurts to include it, but it might cause some confusion for users. Can we ignore it?

Reply:
I would leave it for consistency. Many customers do not uses helm directly (eg. argocd), so I would like to be explicit with all the images